crmsh/utils.py

```diff
@@ -3523,7 +3522,7 @@ def in_haclient():
     """
     Check if current user is in haclient group
     """
-    return constants.HA_GROUP in [grp.getgrgid(g).gr_name for g in os.getgroups()]
+    return constants.HA_GROUP_NUM in list(set(os.getgroups() + [os.getegid()]))
```

Comment:
1. Do not assume the group id of `hacluster` is a constant.
2. No need to convert between set and list.

```suggestion
    return grp.getgrnam(constants.HA_GROUP).gr_gid in (os.getgroups() + [os.getegid()])
```

Reply:
> 1. Do not assume the group id of `hacluster` is a constant.
> 2. No need to convert between set and list.

Still need set to remove duplication

Reply:
We only need to check whether the gid of hacluster is in the list. Duplication in a list does not make a difference in the search result.